java/openmldb-spark-connector/pom.xml

```diff
@@ -69,4 +69,189 @@
         </dependency>
     </dependencies>
 
+    <build>
```

Comment:
those config should inherited from parent pom already ?

Reply:
ok, fixed.